internal/pkg/instrumentation/manager.go

```diff
@@ -398,11 +419,189 @@ func (m *Manager) loadProbes() error {
 	return nil
 }
 
+func (m *Manager) LoadProbe(i ProbeReference, name probe.ID, cfg Config) error {
```

Comment:
`LoadProbe` is only ever called internal to the package. This should be unexported.